src/Compilers/Shared/BuildServerConnection.cs

```diff
@@ -533,8 +540,18 @@ internal static bool WasServerMutexOpen(string mutexName)
                 // the assumption is that it's not open. 
                 return false;
             }
+        }
 
-            return false;
+        internal static IServerMutex OpenOrCreateMutex(string name, out bool createdNew)
+        {
+            if (PlatformInformation.IsRunningOnMono)
+            {
+                return new ServerFileMutexPair(name, true, out createdNew);
```

Comment:
Consider using a named parameter for the second argument here. Lacking that it's unclear what `true` means in this context.